tensorflow/lite/kernels/CMakeLists.txt

```diff
@@ -34,14 +34,16 @@ if(${CMAKE_CROSSCOMPILING})
       ${TFLITE_HOST_TOOLS_DIR}
       ${TFLITE_HOST_TOOLS_DIR}/bin
       ${TFLITE_HOST_TOOLS_DIR}/flatbuffers-flatc/bin
+      ${TFLITE_HOST_TOOLS_DIR}/flatbuffers-build/flatbuffers-flatc/bin
       )
-  find_program(FLATC_BIN flatc PATHS ${FLATC_PATHS})
+  find_program(FLATC_BIN flatc PATHS ${FLATC_PATHS} NO_DEFAULT_PATH)
   if(${FLATC_BIN} STREQUAL "FLATC_BIN-NOTFOUND")
```

Comment:
Why do we have changes without `if (QNX)` guards?

Reply:
Line 37 was introduced to fix an issue with cross compilation that exists in master. Currently tflite uses protobuf but does not support a host build of protoc before building the protobuf libraries. My solution involved adding a protobuf build to the TFLITE_HOST_TOOLS_DIR directory.

The NO_DEFAULT_PATH change was made by Jianyan and I assume because we already explicitly pass in the bin folder. It can be removed if upstream doesn't want it.

Reply:
I guess it's possible they want protoc to be installed to the system before the build so that protoc can be found in PATH, but either way it's strange that the process is different from what's already done for flatbuffers... Oh well, I can always revert that one commit if upstream complains, but I think it's better this way.

Reply:
Especially since tflite already explicitly builds the protoc binaries whether cross-compiling or not...